configs/auto_docking/auto_docking_sim.yaml

```diff
@@ -1,7 +1,8 @@
 auto_docking:
+  differential_drive: True
   kp:
-    x:           0.3
-    y:           1.5
+    # x:           0.3
+    x:           0.6
```

Comment:
Here you replaced the y by x. So now there is no y setting.

Reply:
This is for simulation, the entire module for simulation needs to be adjusted. I will do it over the weekend or maybe next week.